packages/block-editor/src/components/block-alignment-control/use-available-alignments.js

```diff
@@ -46,7 +46,7 @@ export default function useAvailableAlignments( controls = DEFAULT_CONTROLS ) {
 	}
 
 	// Starting here, it's the fallback for themes not supporting the layout config.
-	if ( layoutType.name !== 'default' ) {
+	if ( layoutType.name !== 'default' && layoutType.name !== 'column' ) {
```

Comment:
We should in theory have been able to change this to `if ( layoutType.name !== 'column' )`, but that caused the `BlockAlignmentUI` unit tests to fail, because rendering that component is dependent on a layout type (which defaults to `default`) and in order for tests to succeed it needs to have some alignment controls inside it 😅